node/test/python/aws-lambda-sdk/integration.test.js

```diff
@@ -611,6 +611,12 @@ describe('Python: integration', function () {
   ]);
 
   const useCasesConfig = new Map([
+    [
+      'internal_dependencies',
+      {
+        variants: new Map([['v3-10', { configuration: { Runtime: 'python3.10' } }]]),
+      },
+    ],
```

Comment:
If there's just a single _variant_ I think there's no point to rely on `variants` config

Also shouldn't we use `python3.10` as default runtime for testing now?

Reply:
Good points, amended the PR 👍